src/main/java/seedu/address/logic/commands/UndoCommand.java

```diff
@@ -10,6 +10,9 @@
  */
 public class UndoCommand extends Command {
     public static final String COMMAND_WORD = "undo";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Reverts the person list or task list to a previous state, before an "
+            + "add/delete/edit person or newtask/deltask command was executed";
```

Comment:
": Reverts the person list or task list to a previous state, before an "
+ "add/delete/edit person or newtask/deltask command was executed" + "Example: undo"

Sorry I missed out the example for the command usage for undo and redo. I think adding them in is good and will make it more consistent with the other commands!